cli/src/main/resources/nls/en_US.properties

```diff
@@ -0,0 +1,26 @@
+usage=Usage:
```

Comment:
why do we need such file? IMHO this should be deleted it is also missing the `ide_` prefix and therefore seems to be pointless.

Reply:
I forgot to remove these files because I tried something to fix the last error that was fixed by adding the resources